src/stores/index.ts

```diff
@@ -45,12 +47,13 @@ listenerMiddleware.startListening( {
 export const rootReducer = combineReducers( {
 	chat: chatReducer,
 	wordpressVersions: wordpressVersionsReducer,
+	[ wpComApi.reducerPath ]: wpComApi.reducer,
 } );
 
 export const store = configureStore( {
 	reducer: rootReducer,
 	middleware: ( getDefaultMiddleware ) =>
-		getDefaultMiddleware().prepend( listenerMiddleware.middleware ),
+		getDefaultMiddleware().concat( wpComApi.middleware ).prepend( listenerMiddleware.middleware ),
```

Comment:
```suggestion
		getDefaultMiddleware().prepend( listenerMiddleware.middleware, wpComApi.middleware ),
```
Is the order important here?